libs/service-client/src/Service.php

```diff
@@ -60,6 +62,7 @@ public function getInternalServiceName(): string
             self::IMPORT => 'sapi-importer.default',
             self::NOTIFICATION => 'notification-api.default',
             self::OAUTH => 'oauth-api.default',
+            self::QUERY => 'query-service-api.query-service',
```

Comment:
Why is one some services the suffix `default` and on some there is a name of the service like `query-service`? :)